lib/blockchain_double.js

```diff
@@ -787,6 +796,53 @@ BlockchainDouble.prototype.processTransactionTrace = function(hash, params, call
   });
 };
 
+BlockchainDouble.prototype.processStorageTrace = function(structLog, storageStack, event, callback) {
+  var self = this;
+  var name = event.opcode.name;
+
+  var argsNum = event.opcode.in;
+  var args = event.stack.slice(-argsNum)
+    .map((arg) => to.hex(arg));
+
+  if (storageStack.currentDepth > event.depth) {
+    storageStack.pop()
+  } if (storageStack.currentDepth < event.depth) {
+    storageStack.stack.push({})
+  }
```

Comment:
Just a note even though it's probably not worth being actionable right now: I'd be apprehensive about introducing another stateful variable passed around as reference (or maybe I am just shilling Redux too hard)